modelhub/modelhub/series/series_objectiv.py

```diff
@@ -74,8 +74,11 @@ def get_contexts(self, name: str) -> 'SeriesGlobalContext':
         """
         Returns an array of contexts from an Objectiv stack where `_type` matches the
         UpperCamelCase context and created from `name` e.g. 'identity' becomes 'IdentityContext'.
+
+        @param name:    The name of the context-array to retrieve
```

Comment:
```suggestion
        Returns an array of contexts from an Objectiv stack where `_type` matches the
        UpperCamelCase context created from `name` e.g. if `name` is 'input_value', then this returns all
        contexts where the `_type` field has value 'InputValueContext'.

        :param name: The name of the context-array to retrieve, in snake_case
```

I just did not get that from the original docstring